orm/fields.py

```diff
@@ -100,16 +101,32 @@ def get_column_type(self):
         return sqlalchemy.Boolean()
 
 
-class DateTime(ModelField):
+class AutoAddMixin(ModelField):
+    def __init__(self, auto_now=False, auto_now_add=False, **kwargs):
+        self.auto_now = auto_now
+        self.auto_now_add = auto_now_add
+        if auto_now_add and auto_now:
+            raise ValueError("auto_now and auto_now_add cannot be both True")
+        if auto_now_add or auto_now:
+            kwargs["read_only"] = True
+            kwargs["allow_blank"] = True
```

Comment:
You mean they shouldn't be in __init__ or they shouldn't be exist at all ?

Reply:
I mean I get that `kwargs["read_only"]` is required. But why do you set `kwargs["allow_blank"] = True`?

Reply:
Because allow_blank should make empty field able and for empty field default will be used.
You can see django do the same here.
https://docs.djangoproject.com/en/4.0/ref/models/fields/#datefield

Reply:
Because allow_blank should make empty field able and for empty field default will be used.
You can see django do the same here.
https://docs.djangoproject.com/en/4.0/ref/models/fields/#datefield

Reply:
I think for `auto_now` it makes sense for the field to bu Nullable but for auto_now_add it's always set to default, so why should it allow empty?

Reply:
I think it makes sense that field with default value should allowed to be empty too.

Reply:
Still not sure why we need allow_null=True.
Does it fail the tests?

Reply:
No It's not about tests it just makes sense for me that fields with default value should also allowed to be blank.
But anyway I removed it.